ceno_zkvm/src/instructions/riscv/slti.rs

```diff
@@ -122,9 +122,10 @@ impl<E: ExtensionField, I: RIVInstruction> Instruction<E> for SetLessThanImmInst
                     lkm,
                     *rs1_value.as_u16_limbs().last().unwrap() as u64,
                 )?;
+                let (rs1, imm) = (rs1 as SWord, imm as SWord);
                 config
                     .lt
-                    .assign_instance_signed(instance, lkm, rs1 as SWord, imm as SWord)?;
+                    .assign_instance_signed(instance, lkm, rs1 as i64, imm as i64)?;
```

Comment:
Do you mean changing the inputs in `assign_instance_signed` to `rs1 as SWord as i64` and `imm as SWord as i64` rather than breaking the first conversion out in a separate line?  For me that was just for readability to make it clear on a line that now the values should be interpreted as signed.

(Just to be sure we're on the same page, `rs1 as SWord as i64` is not equivalent to `rs1 as i64` due to the way that sign extension works.)

Reply:
Regarding the use of `into()` in this context, I'm a bit hesitant to use it here just because the way that integer type conversions work is a bit fragile and context-dependent (e.g. in the above there are two different natural ways to interpret a `u32` value as `i64`, depending on whether you step through `i32` first), so I wouldn't want an interface change to implicitly result in using an incorrect conversion.

Reply:
i got the meaning, its u32 -> i32 -> i64 to make signed extension work. so must be this way. thamks for the explaon :)